pom.xml

```diff
@@ -105,6 +105,16 @@
 				<version>0.5.0</version>
 			</dependency>
 			<dependency>
+				<groupId>org.springframework.amqp</groupId>
+				<artifactId>spring-rabbit</artifactId>
+				<version>1.6.2.RELEASE</version>
+			</dependency>
+			<dependency>
```

Comment:
I think that Mockito comes with Spring Boot Test doesn't it? If that's the case you don't have to explicitly provide it here